packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6512,12 +6512,21 @@ export default class Meeting extends StatelessWebexPlugin {
    *
    * @private
    * @static
+   * @param {boolean} isAudioEnabled
+   * @param {boolean} isVideoEnabled
    * @returns {Promise<void>}
    */
-  private static async handleDeviceLogging(): Promise<void> {
+  private static async handleDeviceLogging(
+    isAudioEnabled = false,
+    isVideoEnabled = false
+  ): Promise<void> {
     try {
-      const devices = await getDevices();
+      const deviceKind =
+        (isVideoEnabled && Media.DeviceKind.VIDEO_INPUT) ||
+        (isAudioEnabled && Media.DeviceKind.AUDIO_INPUT) ||
+        undefined;
```

Comment:
@mkesavan13 Im unable to understand this logic. When isVideoEnabled is false and isAudioENabled is false, then shouldnt deviceKind = undefined. And that should call getDevice(undfiened) -> which means we ask for both the permissions.

Reply:
@Parimala032 can answer to this. Logically that is how it is supposed to work, yes